respa_instructions/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ckeditor_uploader.fields import RichTextUploadingField
+
+
+class RespaInstruction(models.Model):
+    ADMIN = 'admin'
+    APPLICANT = 'applicant'
+    APPLICABLE_FOR_CHOICES = (
+        (ADMIN, _('Admin')),
+        (APPLICANT, _('Applicant')),
+    )
```

Comment:
I would move

>     ADMIN = 'admin'
>     APPLICANT = 'applicant'
>     APPLICABLE_FOR_CHOICES = (
>         (ADMIN, _('Admin')),
>         (APPLICANT, _('Applicant')),
>     )
to some config/constants file instead of them being in the class

Reply:
I agree with your reasoning here.  May be is better to have this (moving the constants to a config file) as a separate task as there are other places too using this kind of logic.